src/www/ui/api/Controllers/UserController.php

```diff
@@ -49,6 +50,72 @@ public function getUsers($request, $response, $args)
     return $response->withJson($users, 200);
   }
 
+  /**
+   * Create a user
+   *
+   * @param ServerRequestInterface $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function addUser($request, $response, $args)
+  {
+    $userDetails = $this->getParsedBody($request);
+
+    // creating symphony request
+    $symfonyRequest = new \Symfony\Component\HttpFoundation\Request();
+    $symfonyRequest->request->set('username', $userDetails['name']);
+    $symfonyRequest->request->set('pass1', $userDetails['user_pass']);
+    $symfonyRequest->request->set('pass2', $userDetails['user_pass']);
+    $symfonyRequest->request->set('description', $userDetails['description']);
+    $symfonyRequest->request->set('permission', $this->getEquivalentValueForPermission($userDetails['accessLevel']));
+    $symfonyRequest->request->set('folder', $userDetails['rootFolderId']);
+    $symfonyRequest->request->set('enote', $userDetails['emailNotification'] ? 'y' : 'n');
+    $symfonyRequest->request->set('email', $userDetails['email']);
+    $symfonyRequest->request->set('public', $userDetails['defaultVisibility']);
+    $symfonyRequest->request->set('default_bucketpool_fk', isset($userDetails['defaultBucketpool']) ? $userDetails['defaultBucketpool'] : 2);
+
+    $agents = [
+      "Check_agent_scanoss" => 0,
+      "Check_agent_unpack" => 1,
+      "Check_agent_adj2nest" => 1,
+      "Check_wget_agent" => 1
```

Comment:
These agents can not be chosen by the user. And thus makes no sense to put them in `user_agent_list`

Reply:
@GMishx These changes were included in the patch that you sent. I assumed it to be of some significance so didn't removed it. Should I remove these agents from the list?

Reply:
I am not sure they were in my patch. But please remove them anyways.

The branch can then be merged @krishna9304

Reply:
Sure @GMishx , I'm removing it.